apps/Typescript-ObjC/App.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React, {useEffect} from 'react';
+import type {PropsWithChildren} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  Button,
+  useColorScheme,
+  View,
+} from 'react-native';
+
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+import {
+  CustomerIO,
+  CustomerIOEnv,
+  CustomerioConfig,
+  CioLogLevel,
+} from 'customerio-reactnative';
+
+type SectionProps = PropsWithChildren<{
+  title: string;
+}>;
+
+function Section({children, title}: SectionProps): JSX.Element {
+  const isDarkMode = useColorScheme() === 'dark';
+  return (
+    <View style={styles.sectionContainer}>
```

Comment:
I'll need to learn more about how templates work in this world. I would have expected a file to define the UI template, complete with holders and then a call to a separate function (somewhere) to pass the template and the arguments it requires to fill out any place holders. This feels weird, design-wise, but I don't know what you're using to create the UI to be able to speak much more on it at the moment.

Reply:
This looks like the default template that react native libraries create. I am not sure which package Levi is using to create an app but I have observed similar template for `create-react-native-app` I believe.